.eslintrc.json

```diff
@@ -76,6 +76,22 @@
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/explicit-module-boundary-types": "off"
       }
+    },
+    {
+      "files": ["server/graphql/v2/**/*.+(js|ts)"],
+      "rules": {
+        "no-restricted-imports": [
+          "error",
+          {
+            "patterns": [
+              {
+                "group": ["**/v1/types"],
+                "message": "GraphQL V1 types should not be used with V2."
+              }
+            ]
+          }
+        ]
+      }
```

Comment:
I was thinking while we are at this, shouldn't we do the other way too? Using V2 types being used with V1?

Reply:
That wouldn't bother me. V1 is deprecated and we don't expect the same code quality there than in V2.

Reply:
Yes please